src/test/java/uk/gov/pay/connector/wallets/googlepay/GooglePayServiceTest.java

```diff
@@ -63,6 +65,23 @@ public void shouldAuthoriseAValidCharge() {
         assertThat(authorisationResponse.getEntity(), is(ImmutableMap.of("status", "AUTHORISATION SUCCESS")));
     }
 
+    @Test
+    public void shouldReturnAuthorise3dsRequiedForAValid3dsCharge() {
```

Comment:
Minor spelling error

```suggestion
    public void shouldReturnAuthorise3dsRequiredForAValid3dsCharge() {
```

Reply:
Done.